Makefile.rrw

```diff
@@ -0,0 +1,27 @@
+UNITTESTS_COMMON=test.cc benchmark.cc test_allocator.cc test_blocking_counter.cc test_fixedpoint.cc test_math_helpers.cc

```

Comment:
As discussed earlier, please don't add top-level files, but this can go in contrib/ .